### PR TITLE
client API: add function to simulate input.

### PR DESCRIPTION
**Pull request by torque**
As discussed on IRC a few days ago.

### CODE REVIEW

**Comment by wm4** [comment]
I think this should be a command, not an API function. Depending on what's changed, the addition should be noted in client-api-changes.rst or interface-changes.rst. Other than that, seems ok.

**Comment by torque** [comment]
Hmm, okay, that makes sense. I'll move it to the command interface and make sure it gets documented.

File: player/command.c [hunk 1]
```diff
@@ -4846,6 +4846,37 @@ int run_command(struct MPContext *mpctx, struct mp_cmd *cmd, struct mpv_node *re
...
+        const char *key_name = cmd->args[0].v.s;
+        int code = mp_input_get_key_from_name(key_name);
+        if (code < 0) {
+            MP_ERR(mpctx, "%s is not a valid input name.\n", key_name);
+            return -1;
+        }
+        if (cmd->id == MP_CMD_KEYPRESS)
+            mp_input_put_key(mpctx->input, code);
+        else
+            mp_input_put_key(mpctx->input, code | MP_KEY_STATE_DOWN);
```
Comment (wm4) [inline player/command.c]: I'd just do 
```
if (cmd->id == MP_CMD_KEYPRESS)
    code |= MP_KEY_STATE_DOWN;
```
(Shorter by 1 line and doesn't duplicate the call.)

**Comment by wm4** [comment]
This was merged as d0fe5e08b984db840545c1b341a1e56fad304593 long ago.

**Pull request closed without merging**